chipsec_util.py

```diff
@@ -162,40 +162,49 @@ def parse_switches(self) -> None:
 
     def main(self) -> int:
         """Receives and executes the commands"""
-
         if self._show_banner:
             print_banner(self.argv, get_version(), get_message())
 
-        # @TODO: change later
-        # all util cmds assume 'chipsec_util.py' as the first arg so adding dummy first arg
-        self.argv = ['dummy'] + [self._cmd] + self._cmd_args
-        comm = self.commands[self._cmd](self.argv, cs=self._cs)
-        req_driver = comm.requires_driver()
-        if req_driver and self._no_driver:
-            self.logger.log("Cannot run without driver loaded", level.ERROR)
-            sys.exit(ExitCode.OK)
+        comm = self.commands[self._cmd](self._cmd_args, cs=self._cs)
+        comm.parse_arguments()
+        reqs = comm.requirements()
+        if reqs.load_driver() and self._no_driver:
+            self.logger.log("Cannot run command without a driver loaded.", level.ERROR)
+            return ExitCode.ERROR
+            
+        if reqs.load_config() and not self._load_config:
+            self.logger.log("Cannot run command without a config loaded. Please run with -p and/or --pch if needed.", level.ERROR)
+            return ExitCode.ERROR
 
         try:
-            self._cs.init(self._platform, self._pch, self._helper, ((not self._no_driver) and req_driver), self._load_config, self._ignore_platform)
+            self._cs.init(self._platform, self._pch, self._helper, ((not self._no_driver) and reqs.load_driver()), (self._load_config and reqs.load_config()), self._ignore_platform)
```

Comment:
Do we have precedence between the module declared requirements and the command line arguments? We already did a sanity check in 172 and 175, so here canwe  just use reqs.load_driver() ?